src/main/java/uk/gov/companieshouse/officerfiling/api/validation/OfficerUpdateValidator.java

```diff
@@ -123,10 +126,52 @@ public void validateChangeDateAfterIncorporationDate(HttpServletRequest request,
 
     public void validateChangeDateAfterAppointmentDate(HttpServletRequest request, List<ApiError> errorList, OfficerFilingDto dto, AppointmentFullRecordAPI companyAppointment) {
         var appointmentDate = getAppointmentDate(request, companyAppointment);
-        if(dto.getDirectorsDetailsChangedDate() == null) {
+        if (dto.getDirectorsDetailsChangedDate() == null) {
             createValidationError(request, errorList, apiEnumerations.getValidation(ValidationEnum.CHANGE_DATE_MISSING));
-        } else if(appointmentDate.isPresent() && dto.getDirectorsDetailsChangedDate().isBefore(appointmentDate.get())) {
+        } else if (appointmentDate.isPresent() && dto.getDirectorsDetailsChangedDate().isBefore(appointmentDate.get())) {
             createValidationError(request, errorList, apiEnumerations.getValidation(ValidationEnum.CHANGE_DATE_BEFORE_OFFICER_APPOINTMENT_DATE));
         }
     }
+
+    public void validateNationalitySection(HttpServletRequest request, List<ApiError> errorList, OfficerFilingDto dto, AppointmentFullRecordAPI appointment) {
+        // If hasBeenUpdated boolean is true or null then continue
+        if (Boolean.FALSE.equals(dto.getNationalityHasBeenUpdated())) {
+            return;
+        }
+        // If any of the fields within this section have been provided then continue
+        if (dto.getNationality1() == null && dto.getNationality2() == null && dto.getNationality3() == null) {
+            return;
+        }
+        // If the section matches the current chips data then throw a validation error and don't continue
+        if (doesNationalityMatchChipsData(dto, appointment)) {
+            createValidationError(request, errorList, apiEnumerations.getValidation(ValidationEnum.NATIONALITY_MATCHES_CHIPS_DATA));
+            return;
+        }
+        // Perform validation
+        validateNationality1(request, errorList, dto);
```

Comment:
Should be do this earlier? So error is no nationality1 rather than does not match in CHIPS?

Reply:
I'm not sure I follow sorry Andre. In what scenario? In the scenario where nationality1 has not been supplied then the data can't match what is in chips. In the scenario where the data matches what is in chips then there shouldn't be any validation errors from these four validate methods anyway.

Reply:
We should be checking nationaity1 is present first and only checking same as chips later. Suggest moving the field validation before the same as in chips check.

Reply:
I've coded it according to the scenarios on the ticket. I thought we agreed that those would be final for my story and any future requirements would need to be done separately after you have completed the spike.

Reply:
Decision is do the equals to CHIPS check first.